examples/fault-injection/envoy.yaml

```diff
@@ -41,6 +41,8 @@ static_resources:
                   numerator: 0
                   denominator: HUNDRED
           - name: envoy.filters.http.router
+            typed_config: 
+              "@type": type.googleapis.com/envoy.extensions.filters.http.router.v3.Router
```

Comment:
https://github.com/envoyproxy/envoy/blob/3a44a41879f34f74d78ce8dbed657e8605a5b073/examples/fault-injection/envoy.yaml#L43-L45

In fact, I think the `typed_config` has been added to the example. Have you checked the latest main?